### ISSUE TITLE
Peer reviews are in

### ISSUE THREAD

**Issue by asmeurer** [sympy/sympy-paper#202]
The reviews are in for the paper. There are several revisions that are requested by the referees. 

Question: is everyone able to see them at https://peerj.com/manuscripts/11410/? If not, I will copy them here.

**Comment by aktech** [comment]
> Question: is everyone able to see them at https://peerj.com/manuscripts/11410/?

Yes, I am able to see.

**Comment by leosartaj** [comment]
I am able to see it. Everyone should able to see it though. Should we make somekind of a TODO after reading the comments?

**Comment by asmeurer** [comment]
A checklist for each of the comments would be useful. If someone wants to work on that, feel free to add it to the top of this issue.

**Comment by certik** [comment]
Note that one has to login. So I don't see them from a phone, but hopefully
I'll see them once I login.

**Comment by scopatz** [comment]
Yes, I can see the reviews.

**Comment by certik** [comment]
I can see the reviews now after logging in. Looks like we got very good reviews. We should address all the points that they raised. Regarding the fist reviewer saying it is out of scope --- I thought we submitted to PeerJ Computer Science, but the review question/direction that the reviewer got from the online system seems to suggest we only submitted to PeerJ. @asmeurer, can you clarify this?

**Comment by asmeurer** [comment]
The editor confirmed that it is in scope.

**Comment by certik** [comment]
@asmeurer thanks, that's great news. @asmeurer besides submitting an updated version of the article, how do you submit a reaction to the reviews? Is it just a text, or can it be a pdf document?

Usually one updates the paper and prepares a reaction where each point made by each reviewer is commented on, and explained how the article was updated based on it (if applicable).

**Comment by asmeurer** [comment]
I'm trying to figure that out. I'll forward you the email I got from PeerJ.

**Comment by scopatz** [comment]
There is often also a formal response letter, that the point-by-point messages are either an attachment to or included within

**Comment by certik** [comment]
@asmeurer forwarded me the email from the editor. It specifically asks for a _Rebuttal letter_ (addressing all the Editors' and reviewers' suggestions and feedback, point-by-point) and they provided a link to
[Writing a rebuttal guide](https://peerj.com/benefits/academic-rebuttal-letters/). So that answers my questions.

P.S. I missed the third review (you have to click the "View annotated manuscript" to see it), which asks for a reorganization of the paper. So we should have a discussion how to best do that.

**Comment by asmeurer** [comment]
Yes, I also missed that initially. There are four reviews, one annotated PDF from the editor with a few comments, and three reviews (one of which is an annotated PDF) from the referees.

**Comment by certik** [comment]
@asmeurer ah, and I missed that the editor also reviewed it. So we have 4 reviews.

**Comment by ashutoshsaboo** [comment]
Hi @asmeurer , I am not able to view the reviews. Following is the error message attached, that I receive after I login to my PeerJ account. Is it because my name as a co-author in the paper is not linked to my PeerJ account, or what might be the reason? @certik 

![screenshot image](https://cloud.githubusercontent.com/assets/9900945/17361096/4281a212-598c-11e6-836a-319868f64506.jpeg)

**Comment by asmeurer** [comment]
@ashutoshsaboo it shows that you never accepted the peerj invitation. Did you sign up with the same email that you used for the paper?

**Comment by ashutoshsaboo** [comment]
@asmeurer Yes I did sign up with the same email id (You can also confirm the same here [link](https://github.com/sympy/sympy-paper/blob/master/authors/authors.json) ) Moreover, I use only 1 personal email id, so that's kind of strange. PeerJ didn't send me any mail as well. Can you re-send me the invite somehow to my email id?

**Comment by asmeurer** [comment]
It seems I used ashutosh.saboo@gmail.com instead of ashutosh.saboo96@gmail.com. I've fixed it. Did it send the invite?

**Comment by ashutoshsaboo** [comment]
@asmeurer Ohh, As of now I haven't received the invite. I'll inform as soon as I get the invite. But then you might consider removing the previous email id, i.e  ashutosh.saboo@gmail.com from the co-authorship, because PeerJ must have also sent the invite to that email then?

**Comment by asmeurer** [comment]
I deleted and re-added you. You should have gotten one now.

**Comment by ashutoshsaboo** [comment]
@asmeurer Ah. Still haven't got the invite. :( Some problem with PeerJ or what? Do they instantly send the invite or does it take time?

**Comment by asmeurer** [comment]
I think it doesn't want to send the invite again until we re-submit. I will just send you an email with the reviews.

**Comment by asmeurer** [comment]
The page now also has some technical changes that need to be made as well.

**Comment by asmeurer** [comment]
@ashutoshsaboo there may still be issues. After I resubmitted, PeerJ sent me an email telling me that you still need to accept the invitation. I have contacted them about it. Just a heads up in case they need something from your end.
